react/fields/Autosuggest/Autosuggest.test.js

```diff
@@ -30,7 +32,13 @@ const getAutosuggestProps = (suggestions = []) => ({
 });
 
 describe('Autosuggest', () => {
-  let element, autosuggest, errors, suggestion, suggestionsContainer;
+  let element, autosuggest, errors, suggestions, suggestionsContainer, input;
+
+  const clickSuggestion = suggestion => {
+    Simulate.mouseDown(suggestion);
+    Simulate.mouseUp(suggestion);
+    Simulate.click(suggestion);
```

Comment:
Did you need to do this? This complexity was only added for `ClearField` interaction, you should be able to simply use `Simulate.click`?